carcara/src/parser/mod.rs

```diff
@@ -407,6 +410,68 @@ impl<'a, R: BufRead> Parser<'a, R> {
                 SortError::assert_eq(&Sort::RegLan, sorts[1].as_sort().unwrap())?;
                 SortError::assert_eq(&Sort::String, sorts[2].as_sort().unwrap())?;
             }
+            Operator::BvNot | Operator::BvNeg => {
+                assert_num_args(&args, 1)?;
+                for s in sorts {
+                    let s = s.as_sort().unwrap().clone();
+                    if !matches!(s, Sort::BitVec(_)) {
+                        return Err(ParserError::ExpectedBvSort(s));
+                    }
+                }
+                // TODO: create args value assertion 
+                // assert_args_value(&args, 1..)?;
```

Comment:
Here's the location I meant where I should make args value checking.

https://github.com/ufmg-smite/carcara/pull/22/files#r1394114389